app/javascript/packs/rails_admin.js

```diff
@@ -0,0 +1,2 @@
+import "rails_admin/src/rails_admin/base";
+import "../stylesheets/rails_admin.scss";
```

Comment:
I'd preferably have this entrypoint as a `*.ts` file, if that works. Contents don't need to be modified for that!

Reply:
555f47d